### PR TITLE
sys-fs/ncdu{,-bin}: misc changes

### PR DESCRIPTION
**Pull request by BratishkaErik**

### CODE REVIEW

**Comment by BratishkaErik** [comment]
@jsmolic I'm not feeling right about creating another [disturbing not-expected-when-grabbing-ncdu] package with you as a maintainer just because you maintain the "main" package... If you'd like to, I'd like to co–maintain these packages (sorry for bad English)

File: sys-fs/ncdu/ncdu-2.2.ebuild [hunk 1]
```diff
@@ -21,6 +21,7 @@ RDEPEND="${DEPEND}"
 BDEPEND="
 	|| ( ~dev-lang/zig-0.9.1 ~dev-lang/zig-bin-0.9.1 )
 	virtual/pkgconfig
+	dev-lang/perl
```
Comment (Flowdalic) [inline sys-fs/ncdu/ncdu-2.2.ebuild:24]: The commit subject says "Add missing dependency pod2man" but actually dev-lang/perl is added. Is that right?

Comment (BratishkaErik) [reply sys-fs/ncdu/ncdu-2.2.ebuild:24]: > The commit subject says "Add missing dependency pod2man" but actually dev-lang/perl is added. Is that right?

I checked `equery b /usr/bin/pod2man` and other ebuilds in tree (f.e [app-arch/zpaq/zpaq-7.15-r1.ebuild)](https://github.com/gentoo/gentoo/blob/5bdf7768de4b59c0d93eb49e7d0f65a948820949/app-arch/zpaq/zpaq-7.15-r1.ebuild#L20):
```
equery b /usr/bin/pod2man
 * Searching for /usr/bin/pod2man ... 
dev-lang/perl-5.36.0-r1 (/usr/bin/pod2man-4.140.0-perl-5.36.0)
```

Comment (thesamesam) [reply sys-fs/ncdu/ncdu-2.2.ebuild:24]: Yeah, it sounds odd, but it's correct in this case. Perl has a bunch of binaries you wouldn't expect included in its package.

Comment (Flowdalic) [reply sys-fs/ncdu/ncdu-2.2.ebuild:24]: In that case, maybe change the commit subject to "add missing dev-lang/perl dependency for pod2man".

Comment (BratishkaErik) [reply sys-fs/ncdu/ncdu-2.2.ebuild:24]: > In that case, maybe change the commit title to "add missing dev-lang/perl dependency for pod2man".

Done.

**Review by jsmolic — APPROVED** [review]
Thanks for working on this. Feel free to also add yourself as co-maintainer for these packages (don't worry about adding me as maintainer for new package, it certainly makes sense as it's tied with ncdu)

**Comment by BratishkaErik** [comment]
> Thanks for working on this. Feel free to also add yourself as co-maintainer for these packages (don't worry about adding me as maintainer for new package, it certainly makes sense as it's tied with ncdu)

Okay! :)

**Pull request closed without merging**